goentri.com.fourthwall.json

```diff
@@ -8,102 +8,102 @@
   "description": "Allows user to easily set up domain using Entri",
   "syncPubKeyDomain": "goentri.com",
   "syncRedirectDomain": "api.goentri.com, goentri.com",
-  "variableDescription": "ip is the website server ip, zendeskverification is the text verification value for zendesk, sendgridHost is sendgrid dynamic host.",
+  "variableDescription": "ip is the website server ip, zendeskverification is the text verification value for zendesk, sendgridHost is sendgrid dynamic host., fourthwallHost is the fourthwall host name to be added, supportHost is the support host being added by fourthwall",
```

Comment:
Looking at this template now with variables on 2 different hosts I see even more prominent that these are in fact 2 services.

A website on `%fourthwallHost%` and an email service on `%supportHost%`.
With the use of variables are they are now, instead of `host` parameter, the limitation will be that this template would be able to apply only once per zone, also when `%fourthwallHost%` and `%supportHost%` will be different.
`multiInstance` may be a way to workaround this limitation, however it would not employ this parameter as per its intended use.

An alternative would be to split into separate services for web and email. 2 templates with one consent is currently only possible with async flow, otherwise the sync flow would have to be initiated twice. Async flow was alway considered as the way to go for complex use-cases and therefore more powerful compared to the sync flow.

2 templates would offer also some flexibility, like removal of one without kicking of the other.